carpool-map/css/l.geosearch.css

```diff
@@ -0,0 +1,75 @@
+.displayNone {
```

Comment:
Completed all repathing and using of minified / uglified js files here: https://github.com/voteamerica/voteamerica.github.io/pull/152/commits/c66a5d90eab275898af003ebe912fd334778bc01 and here: https://github.com/voteamerica/voteamerica.github.io/pull/152/commits/09beae90373d678b8d9d09c502300bb6cc541275